techniques/html/authentication.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+
+  <head>
+    <title>Providing a properly marked up email and password inputs</title>
+    <link rel="stylesheet" type="text/css" href="../../css/editors.css" />
+  </head>
+
+  <body>
+    <h1>Providing a properly marked up email and password inputs</h1>
```

Comment:
remove the "a": "Providing *a* properly marked up email and password inputs" > "Providing properly marked up email and password inputs"